From 88abdcf96be2b5c6f9df38ea3009a7b4a1f00078 Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Tue, 25 Oct 2011 18:00:07 +0800
Subject: [PATCH 027/641] UniCore64: Add mm/mmu.c and empty_zero_page

Fixed error:
futex.c:(.text+0x2a2fe): undefined reference to `empty_zero_page'

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/mm/Makefile |    1 +
 arch/unicore64/mm/mmu.c    |    5 +++++
 2 files changed, 6 insertions(+)
 create mode 100644 arch/unicore64/mm/Makefile
 create mode 100644 arch/unicore64/mm/mmu.c

diff --git a/arch/unicore64/mm/Makefile b/arch/unicore64/mm/Makefile
new file mode 100644
index 0000000..1c17518
--- /dev/null
+++ b/arch/unicore64/mm/Makefile
@@ -0,0 +1 @@
+obj-y				:= mmu.o
diff --git a/arch/unicore64/mm/mmu.c b/arch/unicore64/mm/mmu.c
new file mode 100644
index 0000000..a5ec791
--- /dev/null
+++ b/arch/unicore64/mm/mmu.c
@@ -0,0 +1,5 @@
+/*
+ * empty_zero_page is a special page that is used for
+ * zero-initialized data and COW.
+ */
+struct page *empty_zero_page;
-- 
1.7.9.5

